packages/story-editor/src/app/rightClickMenu/constants.js

```diff
@@ -65,63 +62,94 @@ const StyledKbd = styled(Text).attrs({
 `;
 
 export const RIGHT_CLICK_MENU_SHORTCUTS = {
-  SEND_BACKWARD: (
-    <>
-      <StyledKbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</StyledKbd>{' '}
-      <StyledKbd aria-label={SPECIAL_KEYS.DOWN.title}>
-        {SPECIAL_KEYS.DOWN.symbol}
-      </StyledKbd>
-    </>
-  ),
-  SEND_TO_BACK: (
-    <>
-      <StyledKbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</StyledKbd>{' '}
-      <StyledKbd aria-label={SPECIAL_KEYS.SHIFT.title}>
-        {SPECIAL_KEYS.SHIFT.symbol}
-      </StyledKbd>{' '}
-      <StyledKbd aria-label={SPECIAL_KEYS.DOWN.title}>
-        {SPECIAL_KEYS.DOWN.symbol}
-      </StyledKbd>
-    </>
-  ),
-  BRING_FORWARD: (
-    <>
-      <StyledKbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</StyledKbd>{' '}
-      <StyledKbd aria-label={SPECIAL_KEYS.UP.title}>
-        {SPECIAL_KEYS.UP.symbol}
-      </StyledKbd>
-    </>
-  ),
-  BRING_TO_FRONT: (
-    <>
-      <StyledKbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</StyledKbd>{' '}
-      <StyledKbd aria-label={SPECIAL_KEYS.SHIFT.title}>
-        {SPECIAL_KEYS.SHIFT.symbol}
-      </StyledKbd>{' '}
-      <StyledKbd aria-label={SPECIAL_KEYS.UP.title}>
-        {SPECIAL_KEYS.UP.symbol}
-      </StyledKbd>
-    </>
-  ),
-  DELETE: (
-    <StyledKbd aria-label={SPECIAL_KEYS.DELETE.title}>
-      {SPECIAL_KEYS.DELETE.symbol}
-    </StyledKbd>
-  ),
-  COPY_STYLES: (
-    <>
-      <StyledKbd aria-label={optionOrAlt.title}>{optionOrAlt.symbol}</StyledKbd>{' '}
-      <StyledKbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</StyledKbd>{' '}
-      <StyledKbd>{'O'}</StyledKbd>
-    </>
-  ),
-  PASTE_STYLES: (
-    <>
-      <StyledKbd aria-label={optionOrAlt.title}>{optionOrAlt.symbol}</StyledKbd>{' '}
-      <StyledKbd aria-label={cmdOrCtrl.title}>{cmdOrCtrl.symbol}</StyledKbd>{' '}
-      <StyledKbd>{'P'}</StyledKbd>
-    </>
-  ),
+  SEND_BACKWARD: {
+    display: <StyledKbd>{prettifyShortcut('mod down')}</StyledKbd>,
+    title: sprintf(
+      /* Translators: 1: Command/Carat keyboard key.*/
+      _x(
+        '%s down',
+        'The keyboard keys "Command" or "Carat" and "down"',
+        'web-stories'
+      ),
+      getKeyForOS('mod')
+    ),
+  },
+  SEND_TO_BACK: {
+    display: <StyledKbd>{prettifyShortcut('mod shift down')}</StyledKbd>,
+    title: sprintf(
+      /* Translators: 1: Command/Carat keyboard key.*/
+      _x(
+        '%s Shift down',
+        'The keyboard keys "Command" or "Carat", "Shift", and "down"',
```

Comment:
```suggestion
        'The keyboard keys "Command" or "Caret", "Shift", and "down"',
```

😎  `Carat` is the proportion of pure gold, like 24k.